src/components/reminders/reminder.jsx

```diff
@@ -0,0 +1,35 @@
+/** @jsx React.DOM */
```

Comment:
Please remove any of these lines that you see. https://github.com/foundersandcoders/carrier-pigeon/issues/198